src/lints/trait_added_supertrait.ron

```diff
@@ -0,0 +1,58 @@
+SemverQuery(
+    id: "trait_added_supertrait",
+    human_readable_name: "non-sealed trait added new supertraits",
+    description: "A non-sealed trait added one or more supertraits, which breaks downstream implementations of the trait",
+    required_update: Major,
+    lint_level: Deny,
+    reference_link: None, // TODO
```

Comment:
What do you think about citing this portion of the reference here? It's not an exact match, but it's along parallel lines (we're tightening the bounds on which the trait can be implemented) and something is better than nothing...

https://doc.rust-lang.org/cargo/reference/semver.html#generic-bounds-tighten

Reply:
I agree that it's better than nothing, but it may be really confusing. Maybe add a comment on top?

Reply:
I'm open to it! Feel free to add a comment.

Reply:
Couldn't find a way to add value with a comment